README.md
Permission Denied ...

history.txt
    1  apt update
    2  apt upgrade
    3  reboot
    4  pwd
    5  clear
    6  vi /etc/php/7.2/apache2/php.ini
    7  more  /etc/php/7.2/apache2/php.ini
    8  vi /etc/php/7.2/cli/php.ini
    9  sudo service apache2 restart
   10  vi /etc/apache2/sites-enabled/000-default.conf 
   11  vi /etc/apache2/apache2.conf 
   12  systemctl restart apache2
   13  cp /var/lib/pgadmin/storage/postgres_localhost/backuplocalloharano/b_ol_20200413.backup /var/www/html/
   14  exit
   15  cp /var/lib/pgadmin/storage/postgres_localhost
   16  cd /var/lib/pgadmin/storage/postgres_localhost
   17  ls
   18  cp /var/lib/pgadmin/storage/postgres_localhost/persons.csv /var/www/html/
   19  iptables -L
   20  ipetables -A INPUT -p tcp -i etho0 --dport 7014 -j ACCEPT
   21  iptables -A INPUT -p tcp -i etho0 --dport 7014 -j ACCEPT
   22  iptables -L
   23  netstat -pateu | grep LISTEN
   24  service iptables save
   25  /sbin/iptables-save
   26  service apache2 restart
   27  nano /etc/apache2/ports.conf 
   28  cd /etc/apache2/
   29  ls
   30  nano apache2.conf 
   31  cd sites-available/
   32  ls
   33  nano 000-default.conf 
   34  nano default-ssl.conf 
   35  cd ..
   36  cd sites-enabled/
   37  ls
   38  nano 000-default.conf 
   39  cd ..
   40  cd sites-available/
   41  ls
   42  cp default-ssl.conf env-recette.conf
   43  nano env-recette.conf 
   44  a
   45  service apache2 restart
   46  a2ensite env-recette.conf 
   47  service apache2 reload
   48  nano /etc/hosts
   49  nano env-recette.conf 
   50  a2ensite env-recette.conf 
   51  service apache2 reload
   52  nano env-recette.conf 
   53  nano /etc/hosts
   54  nano env-recette.conf 
   55  rm env-recette.conf 
   56  service apache2 reload
   57  sudo netstat -pltn
   58  ipconfig
   59  ifconfig
   60  exit
   61  ifconfig
   62  ping 10.2.250.30
   63  ping 10.2.250.30 80
   64  ping 10.2.250.30:80
   65  telnet 10.2.250.30 80
   66  nping --tcp -p 80 10.2.250.30
   67  nc -vz 10.2.250.30 80
   68  nc -vz 10.2.250.30 2222
   69  nc -vz 10.2.250.30 8069
   70  nc -vz 10.2.250.30 9069
   71  iptables -I INPUT -p tcp --port 8069 -j ACCEPT
   72  iptables -I INPUT -p tcp -m --dport 8069 -j ACCEPT
   73  clear
   74  iptables -I INPUT -p tcp -m tcp --dport 8069 -j ACCEPT
   75  nc -vz 10.2.250.30 8069
   76  ifconfig
   77  iptables -L
   78  nc -vz 192.168.122.1 8069
   79  nc -vz 127.0.0.1 8069
   80  ls /var/www/
   81  rm -rf /var/www/
   82  ls
   83  ls /var/www/
   84  mkdir /var/www/html
   85  mkdir /var/www
   86  mkdir /var/www/html
   87  cd /var/www/html/
   88  cd ../
   89  ls -a
   90  ls -la
   91  chmod 777 html/
   92  chmod 777 -R html/
   93  clear
   94  ls
   95  cd /
   96  ls
   97  cd etc/
   98  ls
   99  sudo /etc/init.d/postgresql status
  100  service postgresql status
  101  clear
  102  service postgresql status
  103  ufw status
  104  iptables -L INPUT
  105  history
  106  cd /var/www/html/
  107  history > history.txt


application/config/ion_auth.php
<?php  if ( ! defined('BASEPATH')) exit('No direct script access allowed');
/**
* Name:  Ion Auth
*
* Version: 2.5.2
*
* Author: Ben Edmunds
*		  ben.edmunds@gmail.com
*         @benedmunds
*
* Added Awesomeness: Phil Sturgeon
*
* Location: http://github.com/benedmunds/CodeIgniter-Ion-Auth
*
* Created:  10.01.2009
*
* Description:  Modified auth system based on redux_auth with extensive customization.  This is basically what Redux Auth 2 should be.
* Original Author name has been kept but that does not mean that the method has not been modified.
*
* Requirements: PHP5 or above
*
*/

/*
| -------------------------------------------------------------------------
| Tables.
| -------------------------------------------------------------------------
| Database table names.
*/
$config['tables']['users']           = 'user';
$config['tables']['groups']          = 'group';
$config['tables']['users_groups']    = 'user_group';
$config['tables']['login_attempts']  = 'login_attempts';

/*
 | Users table column and Group table column you want to join WITH.
 |
 | Joins from users.id
 | Joins from groups.id
 */
$config['join']['users']  = 'user_id';
$config['join']['groups'] = 'group_id';

/*
 | -------------------------------------------------------------------------
 | Hash Method (sha1 or bcrypt)
 | -------------------------------------------------------------------------
 | Bcrypt is available in PHP 5.3+
 |
 | IMPORTANT: Based on the recommendation by many professionals, it is highly recommended to use
 | bcrypt instead of sha1.
 |
 | NOTE: If you use bcrypt you will need to increase your password column character limit to (80)
 |
 | Below there is "default_rounds" setting.  This defines how strong the encryption will be,
 | but remember the more rounds you set the longer it will take to hash (CPU usage) So adjust
 | this based on your server hardware.
 |
 | If you are using Bcrypt the Admin password field also needs to be changed in order to login as admin:
 | $2y$: $2y$08$200Z6ZZbp3RAEXoaWcMA6uJOFicwNZaqk4oDhqTUiFXFe63MG.Daa
 | $2a$: $2a$08$6TTcWD1CJ8pzDy.2U3mdi.tpl.nYOR1pwYXwblZdyQd9SL16B7Cqa
 |
 | Be careful how high you set max_rounds, I would do your own testing on how long it takes
 | to encrypt with x rounds.
 |
 | salt_prefix: Used for bcrypt. Versions of PHP before 5.3.7 only support "$2a$" as the salt prefix
 | Versions 5.3.7 or greater should use the default of "$2y$".
 */
$config['hash_method']    = 'bcrypt';	// sha1 or bcrypt, bcrypt is STRONGLY recommended
$config['default_rounds'] = 8;		// This does not apply if random_rounds is set to true
$config['random_rounds']  = FALSE;
$config['min_rounds']     = 5;
$config['max_rounds']     = 9;
$config['salt_prefix']    = version_compare(PHP_VERSION, '5.3.7', '<') ? '$2a$' : '$2y$';

/*
 | -------------------------------------------------------------------------
 | Authentication options.
 | -------------------------------------------------------------------------
 | maximum_login_attempts: This maximum is not enforced by the library, but is
 | used by $this->ion_auth->is_max_login_attempts_exceeded().
 | The controller should check this function and act
 | appropriately. If this variable set to 0, there is no maximum.
 */
$config['site_title']                 = "Example.com";       // Site Title, example.com
$config['admin_email']                = "admin@example.com"; // Admin Email, admin@example.com
$config['default_group']              = 'members';           // Default group, use name
$config['admin_group']                = 'admin';             // Default administrators group, use name
$config['identity']                   = 'email';             // You can use any unique column in your table as identity column. The values in this column, alongside password, will be used for login purposes
$config['min_password_length']        = 8;                   // Minimum Required Length of Password
$config['max_password_length']        = 20;                  // Maximum Allowed Length of Password
$config['email_activation']           = FALSE;               // Email Activation for registration
$config['manual_activation']          = FALSE;               // Manual Activation for registration
$config['remember_users']             = TRUE;                // Allow users to be remembered and enable auto-login
$config['user_expire']                = 86500;               // How long to remember the user (seconds). Set to zero for no expiration
$config['user_extend_on_login']       = FALSE;               // Extend the users cookies every time they auto-login
$config['track_login_attempts']       = TRUE;                // Track the number of failed login attempts for each user or ip.
$config['track_login_ip_address']     = TRUE;                // Track login attempts by IP Address, if FALSE will track based on identity. (Default: TRUE)
$config['maximum_login_attempts']     = 300;                   // The maximum number of failed login attempts.
$config['lockout_time']               = 600;                 /* The number of seconds to lockout an account due to exceeded attempts
							        You should not use a value below 60 (1 minute) */
$config['forgot_password_expiration'] = 0;                   // The number of milliseconds after which a forgot password request will expire. If set to 0, forgot password requests will not expire.
$config['recheck_timer']              = 0;                   /* The number of seconds after which the session is checked again against database to see if the user still exists and is active.
							           Leave 0 if you don't want session recheck. if you really think you need to recheck the session against database, we would
								   recommend a higher value, as this would affect performance */
								

/*
 | -------------------------------------------------------------------------
 | Cookie options.
 | -------------------------------------------------------------------------
 | remember_cookie_name Default: remember_code
 | identity_cookie_name Default: identity
 */
$config['remember_cookie_name'] = 'remember_code';
$config['identity_cookie_name'] = 'identity';

/*
 | -------------------------------------------------------------------------
 | Email options.
 | -------------------------------------------------------------------------
 | email_config:
 | 	  'file' = Use the default CI config or use from a config file
 | 	  array  = Manually set your email config settings
 */
$config['use_ci_email'] = FALSE; // Send Email using the builtin CI email class, if false it will return the code and the identity
$config['email_config'] = array(
	'mailtype' => 'html',
);

/*
 | -------------------------------------------------------------------------
 | Email templates.
 | -------------------------------------------------------------------------
 | Folder where email templates are stored.
 | Default: auth/
 */
$config['email_templates'] = 'auth/email/';

/*
 | -------------------------------------------------------------------------
 | Activate Account Email Template
 | -------------------------------------------------------------------------
 | Default: activate.tpl.php
 */
$config['email_activate'] = 'activate.tpl.php';

/*
 | -------------------------------------------------------------------------
 | Forgot Password Email Template
 | -------------------------------------------------------------------------
 | Default: forgot_password.tpl.php
 */
$config['email_forgot_password'] = 'forgot_password.tpl.php';

/*
 | -------------------------------------------------------------------------
 | Forgot Password Complete Email Template
 | -------------------------------------------------------------------------
 | Default: new_password.tpl.php
 */
$config['email_forgot_password_complete'] = 'new_password.tpl.php';

/*
 | -------------------------------------------------------------------------
 | Salt options
 | -------------------------------------------------------------------------
 | salt_length Default: 22
 |
 | store_salt: Should the salt be stored in the database?
 | This will change your password encryption algorithm,
 | default password, 'password', changes to
 | fbaa5e216d163a02ae630ab1a43372635dd374c0 with default salt.
 */
$config['salt_length'] = 22;
$config['store_salt']  = FALSE;

/*
 | -------------------------------------------------------------------------
 | Message Delimiters.
 | -------------------------------------------------------------------------
 */
$config['delimiters_source']       = 'config'; 	// "config" = use the settings defined here, "form_validation" = use the settings defined in CI's form validation library
$config['message_start_delimiter'] = '<p>'; 	// Message start delimiter
$config['message_end_delimiter']   = '</p>'; 	// Message end delimiter
$config['error_start_delimiter']   = '<p>';		// Error message start delimiter
$config['error_end_delimiter']     = '</p>';	// Error message end delimiter

/* End of file ion_auth.php */
/* Location: ./application/config/ion_auth.php */
